portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -25,16 +25,24 @@
 /** Servlet that returns  comments from the webpage.*/
 @WebServlet("/comments")
 public class DataServlet extends HttpServlet {
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse 
+    response) throws IOException {
+        // Get the input from the form.
+        String comments = getParameter(request, "comments-container", "");
+        Gson gson = new Gson();
+        String json = gson.toJson(comments);
+        response.setContentType("application/json;");
+        response.getWriter().println(json);
+    }
 
```

Comment:
nit: style guide recommends using 2 space indentation.

Also, I think this and the function declaration below can fit on one line.